uefi/main.c

```diff
@@ -0,0 +1,106 @@
+
+#include <efi.h>
+#include <efilib.h>
+#include <wolfssl/wolfcrypt/logging.h>
```

Comment:
<wolfssl/wolfcrypt/settings.h> (when using user_settings.h otherwise options.h) should always be the first header from wolfSSL included in any example, this is to:

A) Provide good reference to customers using the example
B) Prevent potential undefined behavior

This app doesn't "require it" since logging.h is the only header but it is still a best practice we should keep up.

Other than that no other changes, looks great! Good example and good README.